sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+6. _Constructed Land_, by David Bouchard, Alex Geddie, Bruno Lessard and Pierre Tremblay, was exhibited in 2012 at Interaccess in Toronto. Documentation of the installation, including some of Alex's music, is available at http://nunavutlights.com/. My own Flocking "port" of his SuperCollider instruments, which were foundational to me as both technical exemplars as well as aesthetic ear-benders, are available at https://github.com/colinbdclark/compositions/tree/master/alex-kimmirut
+
+7. I wrote about Flocking's design and architecture in detail in "Flocking: A Framework for Declarative Music-Making on the Web" (Clark and Tindale). The underlying software development idioms and philosophy employed within Flocking and Aconite are described in "Harmonious Authorship from Different Representations" (Basman et al) and in "To Inclusive Design Through Contextually Extended IoC" (Basman et al).
+
+8. Chun, strangely, neglects to examine this issue, choosing instead (as most programmers do) to take the code/data distinction as a given. LISP, however, was the first and paradigmatic language to challenge this; other languages and environments (including my own) have followed. The index of _Programmed Visions_ omits LISP entirely, and Chun limits herself to only a passing reference to John McCarthy and his language, apparently missing the point that it's interactivity wasn't simply a function "of the limitations of procedural programming" (61), but rather a features of its ability to self-reference. For me, the language represents the leak in her schematic regarding the evolution of computation from an emphasis on execution on a specific machine to its erasure by source code. LISP is an environment in which code is represented _as data_, which means that its "source" is changeable by the program itself or by other programs. LISP programs are LISP data structures; this characteristic is known as "homoiconicity," and opens up the possibility that programs, not just data, can be transformed and modified by other programs (Basman et al 255). Complicating the narrative of Chun's argument, LISP fails to maintain the ideology of code as "originary" text, since the materials of a software program is itself available to be modified during the process of interpretation and use.
+
+9. There is a special class of programs, such as parsers, interpreters, code generators, and compilers, which are capable of "reflecting upon" or manipulating traditional non-declarative source code, and which can potentially be used to support programmatic modifiability. However, these tools often do not provide a reliable means for interacting with them and are typically unidirectional; they exemplify, to varying degrees, exactly the kind of erasure that Chun says plagues our models of computation today.
```

Comment:
It may be worth in light of other conversations discussing the specialized programs such as decompilers that can operate (however imperfectly) upon compiled code to try and extract meaning.